tools/deployment/package-helm/templates/redis-statefulset.yaml

```
@@ -0,0 +1,97 @@
+              containerPort: 6379
+          volumeMounts:
+            - name: "redis-data"
+              mountPath: "/data"
+            - name: "redis-logs"
+              mountPath: "/var/log/redis"
+            - name: "config"
+              mountPath: "/etc/redis/redis.conf"
+              subPath: "redis.conf"
+              readOnly: true
+          readinessProbe:
+            {{- include "clp.readinessProbeTimings" . | nindent 12 }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)